Assets/MRTK/Providers/WindowsMixedReality/Shared/WindowsMixedRealityHandMeshProvider.cs

```diff
@@ -6,7 +6,6 @@
 using System;
 
 #if WINDOWS_UWP
-using System.Threading.Tasks;
```

Comment:
Just double checking this will not cause build time error for certain platforms.

Reply:
All good! Just tried another UWP build from Unity 2018 and it succeeded!

![image](https://user-images.githubusercontent.com/3580640/141506625-1582ad4d-4708-460b-ab8c-6ae18842bb8e.png)